config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -60,6 +60,53 @@ presubmits:
           requests:
             memory: "6Gi"
 
+  - name: pull-kubernetes-e2e-gce-canary
+    always_run: false
+    skip_report: true
```

Comment:
always_run: false + skip_report will mean when you `/test pull-kubernetes-e2e-gce-canary` you need to go track down the results yourself /shrug